docs/content/en/install/local/multi-cluster-kustomize/_index.md

```diff
@@ -36,86 +36,89 @@ will run on the same Docker network, which means that they will have routable IP
 Run `setup-kind-multicluster.sh` as follows:
 
 ```sh
-./setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
+scripts/setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
 ```
 
 When creating a cluster, kind generates a kubeconfig with the address of the API server 
 set to localhost. We need a kubeconfig that has the API server address set to its 
 internal ip address. `setup-kind-multi-cluster.sh` takes care of this for us. Generated 
 files are written into a `build` directory.
 
-Run `kubectx` without any arguments and verify that you see the following contexts 
+Run `kubectl config get-contexts` without any arguments and verify that you see the following contexts 
 listed in the output:
 
-* kind-k8ssandra-0
-* kind-k8ssandra-1
+```
+          kind-k8ssandra-0                                            kind-k8ssandra-0                                            kind-k8ssandra-0                                               
+          kind-k8ssandra-1                                            kind-k8ssandra-1                                            kind-k8ssandra-1 
+```
 
 ### Install Cert Manager
 Set the active context to `kind-k8ssandra-0`:
 
 ```console
-kubectx kind-k8ssandra-0
+kubectl config use-context kind-k8ssandra-0
 ```
 
 Install Cert Manager:
 
 ```console
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.9.1/cert-manager.yaml
 ```
 
 Set the active context to `kind-k8ssandra-1`:
 
 ```console
-kubectx kind-k8ssandra-1
+kubectl config use-context kind-k8ssandra-1
 ```
 
 Install Cert Manager:
 
 ```console
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.9.1/cert-manager.yaml
```

Comment:
Should we maybe update this to `1.10.1`? The only reason I ask is that when I go through the Helm install steps, 1.10.1 is the version that gets installed. I don't think it really matters, so if it doesn't, feel free to ignore the comment here.

Reply:
yeah, it'll work either way. We can't update the docs each time they issue a new release ;)